modules/ols-initiating-chat-using-node-options-icon.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is used in the following assemblies:
+// ols-using-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-initiating-chat-using-the-node-options-icon_{context}"]
+= Initiating chat using the Node options icon
+
+This procedure explains how to use the *Node options* icon to inititate a chat using natural language.
+
+.Procedure
+
+. Navigate to a resource in the {ocp-product-title} web console. For example, click *Workloads* -> *Pods* and then click the name of a pod.
+
+. Click the *Node options* icon and select *Ask OpenShift Lightspeed*.
```

Comment:
Is "node options" the 3 dots menu? If that's the case, then you would not click on the name of a pod. You would click the three dots in the row of the desired pod.

Is "node options" something else?

Reply:
Yes @thoraxe it is. 

When I first deployed and tinkered the other day, I thought I had to click on a resource first. I just deployed again and see what you mean. Thanks!